src/test/java/org/jnode/fs/xfs/XfsFileSystemTest.java

```diff
@@ -190,12 +191,44 @@ public void testShortFormAttribute() throws Exception {
             assertThat(attributes, hasSize(1));
             FSAttribute attribute = attributes.get(0);
             assertThat(attribute.getName(), is("selinux"));
-            String stringValue = new String(attribute.getValue(), StandardCharsets.UTF_8)
-                    .replace("\0", "");
+            String stringValue = bytes2string(attribute.getValue());
             assertThat(stringValue, is("unconfined_u:object_r:unlabeled_t:s0"));
         }
     }
 
+    @Ignore("test data not in project, it is 17GB, too large to put in code.")
```

Comment:
Maybe we can set up a mechanism similar to how we handle large test data in Engine. I think it's something like you can declare a parameter which is picked up when running tests, which specifies where a cache of large test data resides.

Reply:
Yeah, we may need some time to do that. I may not support it for now, and will think about it once our feature improvement is done.

Reply:
Just a quick answer, we may not support that, or at least will not do the exactly same thing as engine do.
The reason is we don't want to expose the s3 credentials in code here.

Reply:
I wouldn't think of using S3 necessarily, but possibly just having it so that if the test was run and had a "local" reference to a folder that contained the test data it would run. So yeah, not exactly the same as engine ...
... actually maybe let's forget about this for now.